recipes/linalg-complex/25.10/gh200/compilers.yaml

```diff
@@ -0,0 +1,2 @@
+gcc:
+  version: "14.2"
```

Comment:
gh200 has `version: "14.2"`, while mc variants have `version: "14"`.
If this is necessary, than a comment would help understanding why the compiler is fixed on minor version.

Reply:
Very good point, forgot to add a note about this.

I noticed on the new prgenv-gnu that nvcc fails compilation in some cases with gcc 14.3. In theory they're compatible, but in practice it seems less so. IIRC this particular stack actually compiles with 14.3 as well but I thought it's better to be on the safe side and go with 14.2 like in prgenv-gnu in case some user code would suffer from the same issue.

I'll add this as a comment.